.gitignore

```diff
@@ -7,3 +7,4 @@
 .cproject
 .project
 testing/
+.settings/
```

Comment:
Should this be in the kokkos .gitignore?

Reply:
I think its from @swbova 's editor, and although its not strictly part of this feature, it probably won't hurt Kokkos and should be done eventually (we have `.project` and `.cproject` for similar reasons).